core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -71,6 +72,8 @@ private[spark] class Executor(
 
   private val conf = env.conf
 
+  private val heartbeatInterval = conf.get(EXECUTOR_HEARTBEAT_INTERVAL)
```

Comment:
Just to keep the change narrow and not introduce a field, I think this can stay local to the methods below. For clarity I'd retain the "Ms" suffix on the name.

Reply:
removed